firebase-project/src/views/postList.vue

```diff
@@ -32,23 +45,90 @@
               {{ post.description }}
             </p>
 
-            <p v-if="post.taggedUser" class="mt-2 text-xs text-gray-700">
+            <p
+              v-if="post.taggedUser && post.taggedUser.length"
+              class="mt-2 text-xs text-gray-700"
+            >
               <span class="text-gray-600 font-semibold">Tagged User:</span>
-              <span class="text-gray-800 leading-normal">{{
-                post.taggedUser
-              }}</span>
+              <span
+                class="text-gray-800 leading-normal block"
+                v-for="user in post.taggedUser"
+                :key="user.email"
+                >{{ user.email }}</span
+              >
             </p>
             <p v-else class="mt-2 text-sm text-gray-700 tracking-wide" disabled>
               Tagged User: N/A
             </p>
+          </div>
+          <div class="bg-grey-500">
+            <i
+              class="fa-sharp fa-solid fa-comment text-blue-500 text-lg mr-8 p-2"
+              @click="handleAddComment(post.id)"
+            ></i>
+          </div>
+          <div class="w-96 mt-2 mb-2 flex" v-if="showInput === post.id">
+            <input
+              class="w-48 h-8 ml-4 border-solid border-slate-950 outline-none tracking-wider bg-white p-2 text-sm md:text-base lg:text-lg border-2 md:ml-10 lg:ml-5 xl:w-72 xl:ml-20"
+              name="comment"
+              type="text"
+              placeholder="Add/Update Comment"
+              v-model="comment.commentTitle"
+            />
             <button
-              v-if="posts.length > 0"
-              @click="showPostDetail(post.uid)"
-              class="mt-3 bg-blue-500 hover:bg-blue-700 text-white font-bold text-xs py-1 px-1 rounded"
+              class="bg-blue-500 hover:bg-blue-700 text-white font-bold py-1 px-2 rounded mr-4 ml-2 h-8"
+              @click="submitComment"
             >
-              View Details <i class="fa-solid fa-arrow-right"></i>
+              Submit
             </button>
           </div>
+
+          <div>
+            <ul class="mt-2">
```

Comment:
create a common component for comments.
you can create 2 components
1st for rendering
2nd for edit and add component @mukeshsin